pkg/k8shandler/certificates.go

```diff
@@ -45,6 +50,72 @@ func extractSecretToFile(namespace string, secretName string, key string, toFile
 	return nil
 }
 
+func writeSecret(logging *v1alpha1.ClusterLogging) (err error) {
+
+	secret := utils.Secret(
+		"logging-master-ca",
+		logging.Namespace,
+		map[string][]byte{
+			"masterca":   utils.GetFileContents("/tmp/_working_dir/ca.crt"),
+			"masterkey":  utils.GetFileContents("/tmp/_working_dir/ca.key"),
+			"kibanacert": utils.GetFileContents("/tmp/_working_dir/kibana-internal.crt"),
+			"kibanakey":  utils.GetFileContents("/tmp/_working_dir/kibana-internal.key"),
+		})
+
+	utils.AddOwnerRefToObject(secret, utils.AsOwner(logging))
+
+	err = sdk.Create(secret)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("Failure constructing Curator secret: %v", err)
```

Comment:
Is this error message correct? `Curator`

Reply:
it isn't, good catch